metron-deployment/vagrant/quick-dev-platform/run.sh

```diff
@@ -19,5 +19,5 @@
 
 vagrant \
   --ansible-tags="hdp-deploy,metron" \
-  --ansible-skip-tags="solr" \
+  --ansible-skip-tags="solr,yaf" \
```

Comment:
Yes, I didn't want to install a probe that is not run and has no parser topology.